src/server.rs

```diff
@@ -119,6 +127,23 @@ pub async fn start_with_key(
     }
 
     drop(peer_list);
```

Comment:
is it necessary to drop this ? peer_list is now on global

Reply:
It is not necessary, because it's in a function, and will auto-free. you can delete it.